integration/helpers.go

```diff
@@ -171,6 +171,23 @@ func (this *TeleInstance) GetPortWeb() string {
 	return strconv.Itoa(this.Ports[3])
 }
 
+// GetSiteAPI() is a helper which returns an API endpoint to a site with
+// a given name. This endpoint implements HTTP-over-SSH access to the
+// site's auth server.
+func (this *TeleInstance) GetSiteAPI(siteName string) auth.ClientI {
+	siteTunnel, err := this.Tunnel.GetSite(siteName)
+	if siteTunnel == nil || err != nil {
```

Comment:
1. If `err` == nil this guarantees that `siteTunnel` exists, so this check is redundant